src/fides/api/models/privacy_request.py

```diff
@@ -1812,11 +1808,11 @@ def get_cached_task_id(self) -> Optional[str]:
 
     def get_decoded_access_data(self) -> List[Row]:
         """Decode the collected access data"""
-        return json.loads(self.access_data or "[]", object_hook=_custom_decoder)
+        return self.access_data or []
 
     def get_decoded_data_for_erasures(self) -> List[Row]:
         """Decode the erasure data needed to build masking requests"""
-        return json.loads(self.data_for_erasures or "[]", object_hook=_custom_decoder)
+        return self.data_for_erasures or []
```

Comment:
These functions aren't doing much anymore, but they're still returning the default - maybe they should be renamed -

Reply:
right! i thought about ripping them out altogether but realized the default functionality was nice and important. renaming them seems like a good middle-ground, thanks for the suggestion 👍

Reply:
nice improvement 👍